src/Lean/Elab/MatchExpr.lean

```diff
@@ -112,7 +112,7 @@ Creates a fresh identifier for representing the continuation function used to
 execute the RHS of the given alternative, and stores it in the field `k`.
 -/
 def initK (alt : Alt) : MacroM Alt := withFreshMacroScope do
-  -- Remark: the compiler frontend implemented in C++ currently detects jointpoints created by
+  -- Remark: the compiler frontend implemented in C++ currently detects joinpoints created by
```

Comment:
Should this be "join points"?

@zwarich, I think you're the one to determine our written standard here.

Reply:
On the other hand, it looks like Leo was somewhat consistent about "jointpoint" in this file. There are also a couple of references to "joint point". Probably should get input from @zwarich.

Reply:
Apologies for missing the notification on this. Yes, “join points” is correct.